extensions/apps/extensions/src/components/pages/installed-extensions/default-extensions-list.tsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import getSDK from '@akashaorg/core-sdk';
+import AppList from '@akashaorg/design-system-components/lib/components/AppList';
+import Card from '@akashaorg/design-system-core/lib/components/Card';
+import Stack from '@akashaorg/design-system-core/lib/components/Stack';
+import Text from '@akashaorg/design-system-core/lib/components/Text';
+import Spinner from '@akashaorg/design-system-core/lib/components/Spinner';
+import Button from '@akashaorg/design-system-core/lib/components/Button';
+import ErrorLoader from '@akashaorg/design-system-core/lib/components/ErrorLoader';
+import { useRootComponentProps } from '@akashaorg/ui-awf-hooks';
+import { useGetAppsByPublisherDidQuery } from '@akashaorg/ui-awf-hooks/lib/generated';
+import { selectApps } from '@akashaorg/ui-awf-hooks/lib/selectors/get-apps-by-publisher-did-query';
+import { useTranslation } from 'react-i18next';
+import { type InstalledExtension } from './use-installed-extensions';
+
+export const DefaultExtensionsList = () => {
+  const { t } = useTranslation('app-extensions');
+  const { getCorePlugins, worldConfig } = useRootComponentProps();
+  const sdk = getSDK();
+  const defaultApps = [worldConfig.homepageApp, ...(worldConfig.defaultApps || [])];
+
+  const { data, error, loading } = useGetAppsByPublisherDidQuery({
+    variables: {
+      id: sdk.services.gql.indexingDID,
+      filters: {
+        or: defaultApps.map(app => ({ where: { name: { equalTo: app } } })),
+      },
+      last: defaultApps.length,
```

Comment:
you should add also sorting, otherwise the results might come in different order between calls

Reply:
you could also flip the options to `first: ...` and `sorting: { createdAt: SortOrder.Desc }`, this makes the pagination easier

Reply:
I've changed last to first but the sort order we want it to be ascending so that it starts from Antenna app otherwise it will start from Vibes.